src/client/components/Checker.tsx

```diff
@@ -3,13 +3,22 @@ import { Container, Heading, Divider, Stack, Button } from '@chakra-ui/react'
 import { useForm, FormProvider } from 'react-hook-form'
 
 import { Checkbox, Radio, Numeric } from './fields'
+import { OpArithmetic } from '../operations'
 
 interface CheckerProps {
   config: any
 }
 
+interface Operation {
+  id: string
+  type: string
+  expression: string
+}
+
```

Comment:
Eh, rebase your codebase so that you can pick up some of the latest changes (including the declaration of the Checker type, which should take on board this interface)